### ISSUE TITLE
evaluation loss < training loss

### ISSUE THREAD

**Issue by safooray** [NVIDIA/Megatron-LM#32]
I have pre-trained Megatron's BERT large and base with different batch sizes, and it always seems to be the case that the training loss is about .2 higher than the validation loss. Is this the behavior you observed and if yes, what is causing it?
